roaster-bom/pom.xml

```diff
@@ -38,7 +39,10 @@
 		<version.codeborn.selenide>6.19.1</version.codeborn.selenide>
 
 		<!-- Hibernate -->
-		<version.org.hibernate>6.1.6.Final</version.org.hibernate>
+		<version.org.hibernate>6.2.13.Final</version.org.hibernate>
+
+		<!-- Microprofile -->
+		<version.org.eclipse.microprofile.config>3.0.3</version.org.eclipse.microprofile.config>
```

Comment:
smallrye config kompatibilis ezzel ami épp be van kötve?

Reply:
3.0.2-t húz a smallrye-config-core, szóval igen